### PR TITLE
Bugfix/collect survey import

### LINKED ISSUE #710: Collect Survey Import
- [x] Allow import of .collect file (see #696)
- [x] Import survey description (see #699)
- [ ] Import ApplyIf and Messages (see #705)
- [ ] Import DisplayAs for code attribute (see #736)
- [ ] CollectReportView: add column ```SEVERITY``` (ERROR | WARNING) after ```Type``` (see #704)
- [ ] When collect attribute has a unit specified, add to arena label the unit between square brackets. e.g. Dbh [cm]

### PR DESCRIPTION
**Pull request by SteRiccio**
Fixes #710
- Import Validation Rules - Apply If and Messages
- Import Display As for Code attributes
- CollectReportView: specify validation rules severity (ERROR | WARNING) inside Type column
- When collect attribute has a unit specified, add to arena label the unit between square brackets.

### CODE REVIEW

File: server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js [hunk 1]
```diff
@@ -359,15 +370,12 @@ export default class NodeDefsImportJob extends Job {
...
-        const { condition } = CollectSurvey.getAttributes(element)
-
-        await this.addNodeDefImportIssue(
-          nodeDefUuid,
-          CollectImportReportItem.exprTypes.validationRules,
-          collectExpr,
-          condition,
-          messages,
-        )
+        const { if: condition, flag } = CollectSurvey.getAttributes(element)
```
Comment (SteRiccio) [inline server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js:373]: condition attribute name is "if"

File: server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js [hunk 2]
```diff
@@ -359,15 +370,12 @@ export default class NodeDefsImportJob extends Job {
...
+        const exprType =
+          flag === 'error'
+            ? CollectImportReportItem.exprTypes.validationRuleError
+            : CollectImportReportItem.exprTypes.validationRuleWarning
```
Comment (SteRiccio) [inline server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js:377]: use validation rule flag (severity) to determine expression type (validationRuleError | validationRuleWarning)

File: webapp/loggedin/modules/home/collectImportReport/collectImportReportView.scss [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
+      font-size: 0.8rem;
     }
 
     div {
       white-space: pre-line;
     }
+
+    .icon-warning.warning {
+      color: $orange;
+    }
```
Comment (SteRiccio) [inline webapp/loggedin/modules/home/collectImportReport/collectImportReportView.scss:29]: style for icon warning in Type column

File: webapp/loggedin/surveyViews/labelsEditor/labelsEditor.scss [hunk 4]
```diff
@@ -1,22 +1,20 @@
-@import "~@webapp/style/vars";
+@import '~@webapp/style/vars';
 
 .labels-editor {
-
   display: grid;
-  grid-template-columns: 25px 1fr;
```
Comment (SteRiccio) [inline webapp/loggedin/surveyViews/labelsEditor/labelsEditor.scss:6]: labels editor without labels is used only in Collect Import Report: only one column is needed in the grid template

File: webapp/loggedin/surveyViews/labelsEditor/labelsEditor.scss [hunk 5]
```diff
@@ -1,22 +1,20 @@
-@import "~@webapp/style/vars";
+@import '~@webapp/style/vars';
```
Comment (minotogna) [inline webapp/loggedin/surveyViews/labelsEditor/labelsEditor.scss:1]: we could start using ```@import 'webapp/style/vars';```

Comment (SteRiccio) [reply webapp/loggedin/surveyViews/labelsEditor/labelsEditor.scss:1]: ok

**Pull request merged**